artistools/spectra/plotspectra.py

```diff
@@ -149,17 +149,17 @@ def plot_reference_spectrum(
 
     print_integrated_flux(specdata["f_lambda"], specdata["lambda_angstroms"], distance_megaparsec=metadata["dist_mpc"])
 
-    if len(specdata) > 5000:
-        # specdata = scipy.signal.resample(specdata, 10000)
-        # specdata = specdata.iloc[::3, :].copy()
-        print(f" downsampling to {len(specdata)} points")
-        specdata = specdata.query("index % 3 == 0")
+    # if len(specdata) > 5000:
+    #     # specdata = scipy.signal.resample(specdata, 10000)
+    #     # specdata = specdata.iloc[::3, :].copy()
+    #     print(f" downsampling to {len(specdata)} points")
+    #     specdata = specdata.query("index % 3 == 0")
 
```

Comment:
Why did you take this out? Did it make a difference?

Reply:
Just completely removing two out of every three points of the spectral data was not a great idea. I only did that as a quick hack because my laptop (and software stack) at the time were very slow at handling the AT2017gfo spectra. Plotting performance of those spectra in full resolution is no longer an issue.